tests/ipv4/test_ip_priority_field.py

```diff
@@ -55,11 +43,13 @@ def test_ip_priority_tos(api, b2b_raw_config, utils):
     """
 
     f = b2b_raw_config.flows[0]
-    f.packet = [
-        flow.Header(flow.Ethernet()),
-        flow.Header(flow.Ipv4())
-    ]
-    utils.apply_config(api, b2b_raw_config)
+    # import snappi
```

Comment:
these comments are not needed anymore i suppose, so remove them

Reply:
yes missed to remove it.